src/tree/SlotTreeItem.ts

```diff
@@ -86,7 +91,7 @@ export class SlotTreeItem extends AzExtParentTreeItem implements IProjectTreeIte
     }
 
     public async loadMoreChildrenImpl(_clearCache: boolean, context: IActionContext): Promise<AzExtTreeItem[]> {
-        return await this.resolved.loadMoreChildrenImpl(_clearCache, context);
+        return await this.resolved.loadMoreChildrenImpl.call(this, _clearCache, context) as AzExtTreeItem[];
```

Comment:
Do you think we should just go ahead and use `.call` on all the Impl methods?

Reply:
Since `loadMoreChildrenImpl` is the only method where we need to cast `this as unknown as SlotTreeItem`, I'm pretty confident it's the only place we need to use `.call`. The typings here actually work in our favor and will warn us in the future (we will be forced to cast `this as unknown as SlotTreeItem`).